cibuildwheel/macos.py

```diff
@@ -207,10 +207,6 @@ def setup_python(
         )
         sys.exit(1)
 
-    # Set MACOSX_DEPLOYMENT_TARGET to 10.9, if the user didn't set it.
-    # PyPy defaults to 10.7, causing inconsistencies if it's left unset.
```

Comment:
Actually, pypy is now 10.10, so this is no longer required. We could probably drop the whole thing if we wanted. Though not all builders know to check the version Python was built with and take a maximum of the two.